src/browser-frontend/sagas/tabs-action-handlers.js

```diff
@@ -16,10 +16,11 @@ import { takeEvery, call, put } from 'redux-saga/effects';
 import PagesModelActions from '../actions/pages-model-actions';
 import UIPageModel from '../model/ui-page-model';
 
-function* closeTabAnimated({ payload: { pageId, removePageAfterMs } }) {
+function* closeTabAnimated({ payload: { pageId, removePageAfterMs, selected } }) {
```

Comment:
It's better not to pass in state inside these sagas, but instead query any relevant state inside them. To make business logic more easily testable in the future, the method signature should be as simple as possible, so that this doesn't depend on external callers passing in the "correct" stuff (e.g, this `selected` attribute). While in this particular case the difference would be marginal, and we don't even have tests yet, being consistent across the codebase would be a good idea.

```js
import UIPagesSelectors from '../selectors/ui-pages-selectors';
...
const selectedPageId = yield select(UIPagesSelectors.getSelectedPageId);
const tabState = selectedPageId === pageId 
  ? UIPageModel.TAB_STATES.SELECTEDCLOSED 
  : UIPageModel.TAB_STATES.CLOSED;
```